outputs.tf

```diff
@@ -24,12 +24,12 @@ output "key_protect_instance_policies" {
 
 output "key_rings" {
   description = "IDs of new Key Rings created by the module"
-  value       = module.key_protect_key_rings
+  value       = module.kms_key_rings
 }
 
 output "keys" {
   description = "IDs of new Keys created by the module"
-  value       = module.key_protect_keys
+  value       = module.kms_keys
 }
```

Comment:
hmm, I wonder if the `existing_key_ring_keys` output should now also be part of the `keys` output since the user now passes all keys in using var.keys. 1 input and 1 output probably makes sense here?